{
	"id": "openSUSE-SU-2019:2698-1",
	"modified": "2019-12-22T05:12:51Z",
	"published": "2019-12-22T05:12:51Z",
	"related": [
		"CVE-2019-2737",
		"CVE-2019-2739",
		"CVE-2019-2740",
		"CVE-2019-2758",
		"CVE-2019-2805",
		"CVE-2019-2938",
		"CVE-2019-2974"
	],
	"summary": "Security update for mariadb",
	"details": "This update for mariadb to version 10.2.29 fixes the following issues:\n\nMariaDB was updated to 10.2.29 (bsc#1156669) \n\nSecurity issues fixed: \n\n- CVE-2019-2737: Fixed an issue where could lead a remote attacker to cause denial of service \n- CVE-2019-2938: Fixed an issue where could lead a remote attacker to cause denial of service \n- CVE-2019-2740: Fixed an issue where could lead a local attacker to cause denial of service\n- CVE-2019-2805: Fixed an issue where could lead a local attacker to cause denial of service\n- CVE-2019-2974: Fixed an issue where could lead a remote attacker to cause denial of service\n- CVE-2019-2758: Fixed an issue where could lead a local attacker to cause denial of service\n  or data corruption\n- CVE-2019-2739: Fixed an issue where could lead a local attacker to cause denial of service \n  or data corruption\n\nThis update was imported from the SUSE:SLE-15:Update update project.",
	"affected": [
		{
			"package": {
				"ecosystem": "openSUSE:Leap 15.1",
				"name": "mariadb",
				"purl": "purl:rpm/suse/mariadb&distro=openSUSE%20Leap%2015.1"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "10.2.29-lp151.2.9.1"
						}
					]
				}
			],
			"ecosystem_specific": {
				"binaries": [
					{
						"libmysqld-devel": "10.2.29-lp151.2.9.1",
						"libmysqld19": "10.2.29-lp151.2.9.1",
						"mariadb": "10.2.29-lp151.2.9.1",
						"mariadb-bench": "10.2.29-lp151.2.9.1",
						"mariadb-client": "10.2.29-lp151.2.9.1",
						"mariadb-errormessages": "10.2.29-lp151.2.9.1",
						"mariadb-galera": "10.2.29-lp151.2.9.1",
						"mariadb-test": "10.2.29-lp151.2.9.1",
						"mariadb-tools": "10.2.29-lp151.2.9.1"
					}
				]
			}
		}
	],
	"references": [
		{
			"type": "ADVISORY",
			"url": "https://lists.opensuse.org/archives/list/security-announce@lists.opensuse.org/thread/EL25YKJX6ZC6Q322TAJNA7AK5OZSDX2O/#EL25YKJX6ZC6Q322TAJNA7AK5OZSDX2O"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.suse.com/1156669"
		},
		{
			"type": "WEB",
			"url": "https://www.suse.com/security/cve/CVE-2019-2737"
		},
		{
			"type": "WEB",
			"url": "https://www.suse.com/security/cve/CVE-2019-2739"
		},
		{
			"type": "WEB",
			"url": "https://www.suse.com/security/cve/CVE-2019-2740"
		},
		{
			"type": "WEB",
			"url": "https://www.suse.com/security/cve/CVE-2019-2758"
		},
		{
			"type": "WEB",
			"url": "https://www.suse.com/security/cve/CVE-2019-2805"
		},
		{
			"type": "WEB",
			"url": "https://www.suse.com/security/cve/CVE-2019-2938"
		},
		{
			"type": "WEB",
			"url": "https://www.suse.com/security/cve/CVE-2019-2974"
		}
	]
}
